localgovdrupal/localgov_microsites_base
# Fix/193 sub menu hover

Closes #193 

## What does this change?

Removes hover functionality, so you must explicitely click the toggle button to activate sub menus.

## How to test
Create a microsite with nested sub menus, check that you can access all sub menu items.
